### PR TITLE
Added CSP whitelist policies for avarda-hosted resources

### PR DESCRIPTION
**Pull request by vmalyk**
Hi,

Added 1 additional configuration with CSP policies in [Magento](https://developer.adobe.com/commerce/php/development/security/content-security-policies/) and add control on module.

Please review,

### CODE REVIEW

File: etc/csp_whitelist.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            <values>
+                <value id="avarda-checkout" type="host">*.checkout-api.avarda.com</value>
+            </values>
+        </policy>
+        <policy id="font-src">
+            <values>
+                <value id="avarda-checkout" type="host">*.checkout-api.avarda.com</value>
+            </values>
+        </policy>
+        <policy id="font-src">
```
Comment (joelvai) [inline etc/csp_whitelist.xml:20]: Duplicate policy for the font-src.

**Review by joelvai — CHANGES_REQUESTED** [review]

File: etc/csp_whitelist.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            <values>
+                <value id="avarda-checkout" type="host">*.checkout-api.avarda.com</value>
+            </values>
+        </policy>
+    </policies>
+</csp_whitelist>
```
Comment (joelvai) [inline etc/csp_whitelist.xml:26]: Add empty row to the end.

**Review by joelvai — CHANGES_REQUESTED** [review]

File: etc/csp_whitelist.xml (hunk 1 above)
Comment (vmalyk) [reply etc/csp_whitelist.xml]: Fixed, it should be img-src.

File: etc/csp_whitelist.xml (hunk 2 above)
Comment (vmalyk) [reply etc/csp_whitelist.xml]: Added

**Pull request merged**